semtypes/src/main/java/io/ballerina/types/typeops/DecimalOps.java

```diff
@@ -0,0 +1,50 @@
+package io.ballerina.types.typeops;
```

Comment:
Missing license header. 
You can add this to Java template in your IDE so that you don't have manually add this.